test/test_uri_spec.py

```diff
@@ -23,44 +25,74 @@
 from pymongo.uri_parser import parse_uri
 from test import unittest
 
-# Location of JSON test specifications.
-_TEST_PATH = os.path.join(
+
+CONN_STRING_TEST_PATH = os.path.join(
     os.path.dirname(os.path.realpath(__file__)),
     os.path.join('connection_string', 'test'))
 
+URI_OPTIONS_TEST_PATH = os.path.join(
+    os.path.dirname(os.path.realpath(__file__)), 'uri_options')
+
+TEST_DESC_SKIP_LIST = [
+    "Valid options specific to single-threaded drivers are parsed correctly",
```

Comment:
Depending on how you sync the files from the specifications repo, it might be easier to just not include this file at all in your repo.

Reply:
Fair enough! If you're interested, a while back we wrote a script to handle updating spec tests that you might be interested in using in the future: https://github.com/mongodb/mongo-go-driver/blob/master/etc/update-spec-tests.sh